pkg/controller/llmisvc/DEV.md

```diff
@@ -418,16 +423,130 @@ spec:
 EOF
 ```
 
-**Install OSSM(TBD)**
+**Install OSSM**
+
+You have to add pullsecret for brew image on your cluster.
+
+
+```shell
+
+# Update PULL SECRET
+export BREW_PULL_SECRET_FILE="path/to/file"
+export REGISTRY_PULL_SECRET_FILE="path/to/file"
+
+oc get secret pull-secret -n openshift-config -o jsonpath='{.data.\.dockerconfigjson}' | base64 -d > /tmp/pull-secret.json 
+jq -s '.[0].auths += .[1].auths | {auths: .[0].auths}' /tmp/pull-secret.json $BREW_PULL_SECRET_FILE > /tmp/new-pull-secret.json    
+jq -s '.[0].auths += .[1].auths | {auths: .[0].auths}' /tmp/new-pull-secret.json  $REGISTRY_PULL_SECRET_FILE > /tmp/final-pull-secret.json    
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/tmp/final-pull-secret.json  
+
+# Create MirrorSet to pull prebuilt images 
+cat <<EOF| oc create -f -
+apiVersion: config.openshift.io/v1
+kind: ImageTagMirrorSet
+metadata:
+    name: stage-registry
+spec:
+    imageTagMirrors:
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh
+          source: registry.redhat.io/openshift-service-mesh
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-tech-preview
+          source: registry.redhat.io/openshift-service-mesh-tech-preview
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-dev-preview-beta
+          source: registry.redhat.io/openshift-service-mesh-dev-preview-beta
+---
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+    name: stage-registry
+spec:
+    imageDigestMirrors:
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh
+          source: registry.redhat.io/openshift-service-mesh
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-tech-preview
+          source: registry.redhat.io/openshift-service-mesh-tech-preview
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-dev-preview-beta
+          source: registry.redhat.io/openshift-service-mesh-dev-preview-beta
+EOF
+
+# Deploy OSSM
+cat<<EOF |oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: istio-catalog
+  namespace: openshift-marketplace
+spec:
+  displayName: istio-catalog
+  image: brew.registry.redhat.io/rh-osbs/iib:1015285
```

Comment:
I assume this image is causing the problem? Do we want to wait with these steps before we know the healthy one?